src/app/api/product-reviews/route.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  getProductReviewByUser,
+  getProductReviews,
+  insertReview,
+} from "@/model/ProductReviewsModel";
+import { NextResponse } from "next/server";
+
+export async function POST(request: Request) {
+  let body = undefined;
+  try {
+    body = await request.json();
+  } catch (err) {
+    return NextResponse.json(
+      {
+        message: "Incorect format",
+      },
+      {
+        status: 400,
+      }
+    );
+  }
+  if (
+    typeof body.product_id !== "string" ||
+    typeof body.user_id !== "string" ||
+    typeof body.product_name !== "string" ||
+    typeof body.review !== "string" ||
+    typeof body.rating !== "string"
+  ) {
+    return NextResponse.json(
+      {
+        message: "Incorect format",
+      },
+      {
+        status: 400,
+      }
+    );
+  }
+
+  if (body.rating > "5") {
```

Comment:
This should have been a BIG red flag when developing! Having a `>` between strings is not ok.

The rating should be a number. See my previous comment.